webui/src/components/CXL-Hosts/CXL-Hosts.vue

```diff
@@ -1127,6 +1127,23 @@ export default {
       async (newHostId, oldHostId) => {
         if (newHostId !== null && newHostId !== oldHostId) {
           loading.value = true;
+
+          await hostStore.fetchHostById(newHostId);
```

Comment:
Should you also remove the .fetchHostById() call on line 1152 of this file?

Reply:
yes, you are right, thanks